packages/cerebral/src/Module.js

```diff
@@ -1,35 +1,24 @@
 class Module {
-  constructor (path, module, controller) {
-    this.name = path.slice().pop()
-    this.path = path
-    this.controller = controller
-    this.modules = {}
+  constructor (controller, path, moduleDescription) {
+    const moduleStub = {
+      controller,
+      path: path.join('.'),
+      name: path.slice().pop()
+    }
 
-    const moduleDescription = typeof module === 'function' ? module(this) : module
+    const module = typeof moduleDescription === 'function' ? moduleDescription(moduleStub) : moduleDescription
 
-    this.initialState = moduleDescription.state || {}
-    if (moduleDescription.state) {
-      this.controller.getModel().set(this.path, moduleDescription.state)
-    }
+    module.initialState = module.state || {}
+    controller.getModel().set(path, module.initialState)
```

Comment:
@christianalfoni 
I see no usage of module.initialState.
Could we replace it with just:
```js
controller.getModel().set(path, module.state || {})
```